src/Presentation/Console/Commands/InitProjectCommand.php

```diff
@@ -130,10 +139,6 @@ protected function initializeSettingValues(array $settingEntities, bool $needsTo
                 default => (string)$values,
             };
 
-            if ($settingEntity->getStrategy() === SettingInterface::STRATEGY_MERGE) {
```

Comment:
why did you remove this strategy?

Reply:
Because we have the same code in `projectSettingManager::setSettings`